### ISSUE TITLE
If protocol suffix is alloed for  Grpc default value https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/

### ISSUE THREAD

**Issue by michaelkira** [open-telemetry/opentelemetry.io#4608]
**What needs to be changed?** Describe the update that is required.
In  https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/, the default value for gRPC is with protocol suffix
But such endpoint is dont accepted by telegraf, more details can be found in this issue https://github.com/influxdata/telegraf/issues/15450

Please review protocol suffix is allowed(or disallowed) for the gRPC service

**What is the name + path of the page that needs changed?** The relative path
and page title where you found a problem.
If protocol suffix is alloed for  Grpc default value https://opentelemetry.io/docs/languages/sdk-configuration/otlp-exporter/

**Additional context**: anything else you can think about that would be helpful.

_Please delete paragraphs that you did not use before submitting._

**Comment by theletterf** [comment]
@open-telemetry/collector-approvers Is this something that would need to be changed?

**Comment by mx-psi** [comment]
I don't think this belongs to the Collector SIG, this is about the SDKs OTLP exporter

**Comment by cartermp** [comment]
I'm not sure if this is a bug, but I do think it's probably worth changing. The spec also uses `http` in examples for gRPC endpoints, but I don't know if that was intentional or not, and that's what the SDK config docs are based on.

**Comment by michaelkira** [comment]
@svrnm Thanks, please help transfer. I think this worth addressing or at least a clear classification
